dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23JSF22SingletonFeatureTest.java

```diff
@@ -40,6 +41,7 @@
  */
 @Mode(TestMode.FULL)
 @RunWith(FATRunner.class)
+@SkipForRepeat(SkipForRepeat.EE9_FEATURES)
```

Comment:
If we're skipping this b/c it does not work lets get an issue opened for investigation so we don't forget about it.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/13890